geoportal/c2cgeoportal_geoportal/scaffolds/create/bin/eval-templates

```diff
@@ -1,7 +1,10 @@
 #!/bin/bash -e
 
-find /etc /mapcache -name '*.tmpl' -print | while read file
+find /etc /mapcache /usr/local/tomcat/webapps/ -name '*.tmpl' -print | while read file
 do
+    export VISIBLE_WEB_HOST_RE_ESCAPED=`python -c "print(__import__('re').escape('${VISIBLE_WEB_HOST}'))"`
+    export VISIBLE_ENTRY_POINT_RE_ESCAPED=`python -c "print(__import__('re').escape('${VISIBLE_ENTRY_POINT}'))"`
```

Comment:
Detail, but this could be done once, before the loop.

Reply:
For me it introduce too may complexity just for 2 variables ...

Reply:
Misunderstood, done :-)